dev/com.ibm.ws.security.fat.common.jwt/src/com/ibm/ws/security/fat/common/jwt/expectations/JwtApiExpectation.java

```diff
@@ -0,0 +1,139 @@
+                validationValue = buildClaimApiString(prefix, key, value);
+                break;
+
+            case CLAIM_LIST_MEMBER:
+                if (value == null || value.equals("-1")) {
+                    checkType = JwtConstants.STRING_DOES_NOT_MATCH;
+                } else {
+                    checkType = JwtConstants.STRING_MATCHES;
+                }
+                validationValue = buildJsonClaimString(prefix, key, value);
+                break;
+
+            case CLAIM_FROM_LIST:
+                if (value == null || value.equals("-1")) {
+                    checkType = JwtConstants.STRING_DOES_NOT_MATCH;
+                } else {
+                    checkType = JwtConstants.STRING_MATCHES;
+                }
+                validationValue = buildJsonAllClaimString(prefix, JwtConstants.JWT_JSON + JwtConstants.JWT_GETALLCLAIMS, key, value);
+                break;
+            case HEADER_CLAIM_FROM_LIST:
+                if (value == null || value.equals("-1")) {
+                    checkType = JwtConstants.STRING_DOES_NOT_MATCH;
+                } else {
+                    checkType = JwtConstants.STRING_MATCHES;
+                }
+                validationValue = buildJsonAllClaimString(prefix, JwtConstants.JWT_TOKEN_HEADER_JSON, key, value);
+                break;
+
+            default:
+                break;
+        }
+    }
+
+    public String buildClaimApiString(String prefix, String keyLogName, Object value) {
+
+        String builtString = prefix + keyLogName + ".*" + value;
+
+        return builtString;
+    }
+
+    public String buildJsonClaimString(String prefix, String key, Object value) {
+
+        String builtString = "garbage";
+        String newValue = "garbage";
+        if (value == null) {
+            newValue = ":" + ".*null";
+        } else {
+            if (value.equals("-1")) {
+                newValue = ":";
+            } else {
+                newValue = value.toString().replace("[", "").replace("]", "");
+            }
+        }
+        builtString = prefix + JwtConstants.JWT_JSON + "\\{" + ".*" + key + ".*" + newValue + ".*\\}";
+        return builtString;
+    }
+
+    public String buildJsonAllClaimString(String prefix, String subPrefix, String key, Object value) {
+
+        String builtString = "garbage";
+        if (value instanceof Long) {
+            if (value.equals(-1L)) {
+                builtString = prefix + subPrefix + ".*" + key + ".*";
+            }
+        }
+        builtString = prefix + subPrefix + ".*" + key + ".*" + value;
+
+        return builtString;
+    }
+
+    /*
+     * (non-Javadoc)
+     *
+     * @see com.ibm.ws.security.fat.common.expectations.Expectation#validate(java.lang.Object)
+     */
+    @Override
+    protected void validate(Object contentToValidate) throws Exception {
+//        Log.info(thisClass, "validate (in JwtClaimExpectations", checkType);
+        super.validate(contentToValidate);
```

Comment:
There really isn't a point in having this override method if all it does is call the super.

Reply:
removing with issue 10532